maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -490,11 +491,11 @@
 
     /**
      * Allows you to specify the name of the JUnit Platform artifact.
-     * If not set, {@code org.junit.platform:junit-platform-engine} will be used.
+     * If not set, {@code org.junit.platform:junit-platform-commons} will be used.
```

Comment:
Projects that only refer to `junit-jupiter-api` no longer have `junit-platform-engine` in their dependencies. So now, we are using `junit-platform-commons` as the trigger value to activate the JUnit Platform Provider.